src/duckling/canvas/drawing/drawn-construct.ts

```diff
@@ -1,13 +1,9 @@
-import {Container, DisplayObject, Point} from 'pixi.js';
+import {Container, DisplayObject, Point, Graphics} from 'pixi.js';
```

Comment:
fixed, though we need to look at performance because it's slower to rebuild the display object snow